src/components/Typography.js

```diff
@@ -27,15 +27,15 @@ export const H3 = styled.h3`
 
 export const P = styled.p`
   ${text};
-  ${p => (p.highlight && 'color: ' + p.theme.colors.link)};
+  ${p => (p.highlight && 'color: ' + p.theme.colors.highlight)};
```

Comment:
can we keep this consistent with the other ones and use template literals here too